scripts/common.py

```diff
@@ -19,13 +19,17 @@
 import contextlib
 import getpass
 import os
+import platform
 import re
 import socket
 import subprocess
 
 import python_utils
 import release_constants
 
+NODE_VERSION = '10.15.3'
+YARN_VERSION = 'v1.17.3'
```

Comment:
We should probably not have one version starting with "v" and the other not -- can we standardize?

Following the other folders/libs I guess we should get rid of the "v".

OK to do this in separate PR (but please file an issue if so and put a TODO here linking to it so that other devs know not to follow this pattern).

Reply:
Updated